CONTRIBUTING.md

```diff
@@ -357,8 +363,8 @@ These types of PRs should only involve well-polished and ready for the public ma
 
 **Scenario 1: All changes from staging should be made live**  
 
-- Create a new branch from the most up-to-date `master` branch.
-- Checkout your newly created branch.
+- Create a new branch from the most up-to-date `master` branch, call it `make-it-live`.
```

Comment:
I think we should encourage specificity, something like: `make-live-mychanges`

This will also prevent errors if there are multiple branches with this name and the last one hasn't been deleted yet.